src/runtime/Makefile

```diff
@@ -132,7 +132,9 @@ ROOTMEASURECONFIGTDX ?= ""
 AGENT_AA_KBC_PARAMS ?= ""
 AGENT_AA_KBC_PARAMS_TDX ?= ""
 AGENT_AA_KBC_PARAMS_SEV ?= ""
-TDXKERNELPARAMS := tdx_disable_filter $(ROOTMEASURECONFIGTDX) agent.enable_signature_verification=false $(AGENT_AA_KBC_PARAMS_TDX)
+TDXKERNELPARAMS := tdx_disable_filter agent.enable_signature_verification=false $(AGENT_AA_KBC_PARAMS_TDX)
+TDXKERNELPARAMS_QEMU += $(TDXKERNELPARAMS) $(ROOTMEASURECONFIGTDX)
```

Comment:
This is a bit bike shedding probably, but I just want to clarify my understanding - Does the face that the  `TDXKERNELPARAMS_CLH` use the 'vanilla' root_hash mean that it's more tied to the KBC (or at least the combination of CC HW & KBC, rather than just the CC HW like `ROOTMEASURECONFIGTDX` would make me think, or have I got the logic wrong here?

Reply:
Not really.  The main difference here is that we use one rootfs (the default one) for Cloud Hypervisor, as attestation is not implemented on its side yet; and a different rootfs (the TDX specific one) for QEMU.  That's me main reason for actually having to have those two different variables.

Makes sense?

Reply:
OK, so is the specific difference in the rootfs (and therefore the hash) the attestation-agent KBC that is built in, the VMM, or something else?

Reply:
The specific differente is in the rootfs, and that happens as for TDX we install rats-tls in the rootfs (see: a1571721ddc847d654d24c8ce6a13c5b4f70d203) and also add a default file for using the QGS (see: 68956ad1270511953e841c162867cd511b82c89c).

As those end up being specific to the KBC being used, we end up with two different images, thus two different root hashes.

To complicate things slightly more we noticed that CLH does not work with verdictd (due to the attestation bits missing), thus we do need to keep those 2 different rootfs for TDX, at least for now.

Reply:
Once CLH works with verdictd I'm guessing that it will switch to using `ROOTMEASURECONFIGTDX` and then my confusion will all go away anyway?

Reply:
Exactly! :-)

By the moment CLH works with verdictd we'll stop using the `cc-rootfs-image` for the TDX related configurations, relying only in the `cc-tdx-rootfs-image`, thus on the `ROOTMEASURECONFIGTDX` parameter.

Sorry, this is confusing, I know and I feel your pain.

Reply:
Cool, I think that gives enough info to resolve this thread. Thanks!